proxysql_tools/managers/proxysql_manager.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, host, port, user, password, socket=None,
         :param reload_runtime: Whether the ProxySQL runtime should be
             reloaded for the changes to take affect.
         :type reload_runtime: bool
+        :raise: ProxySQLAdminConnectionError
```

Comment:
`__init__` does not raise this exception. Perhaps you want to add it to the docstring of `ping` function.